content/blog/2017/2017-01-18-refactoring-conditional-to-pipeline.adoc

```diff
@@ -0,0 +1,475 @@
+---
+layout: post
+title: "Refactoring Conditional BuildSteps into Jenkins Pipeline"
```

Comment:
@orrc - @rtyler said "Also, I'm a fan of the use of "upgrading" rather than "migrating" but that's your call"

@daniel-beck - I can do "Converting".